modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad/src/main/java/com/liferay/dynamic/data/mapping/uad/display/DDMFormInstanceUADDisplay.java

```diff
@@ -83,9 +83,9 @@ public DDMFormInstance getTopLevelContainer(
 				DDMFormInstanceRecord ddmFormInstanceRecord =
 					(DDMFormInstanceRecord)childObject;
 
-				if ((long)parentContainerId ==
+				if (parentContainerId.equals(
```

Comment:
@marcosapmf, the `equals` function will work for sure in this case? From what I understood from Brian's comment, you can cast `parentContainerId` but you shouldn't do it together with the operator `==`, did you have the same understanding than me?

Reply:
Hey @natocesarrego!
I had the same understanding than you about Brian's comment. It will work on this way because of it is expected that parentContainerId be a `Long` :)